File: Modul 4/Soal 5/PRAK305-2210817110002-MUHAMMAD PASHA NABEEL.c
#include <stdio.h>

int main()
{
    int sec, hr, jm, mnt, dtk;
    scanf("%d", &sec);
    if(sec<=86400){
        jm =sec/3600;
        dtk=sec-3600*jm;
        mnt=dtk/60;
        dtk=dtk-60*mnt;
        printf("%.2d:%.2d:%.2d\n\n", jm, mnt, dtk);}
    else{
        hr =sec/86400;
        jm =(sec-(hr*86400))/3600;
        mnt=((sec-hr*86400)-(jm*3600))/60;
        dtk=((sec-hr*86400)-(jm*3600))-(mnt*60);
        printf("%d hari %.2d:%.2d:%.2d\n\n", hr, jm, mnt, dtk);}
        return 0;
    }

File: Modul 4/Soal 5/PRAK305-2210817110002-MUHAMMAD PASHA NABEEL.py
sec = int(input())
if sec <= 86400:
    jam   = int(sec/3600)
    detik = int(sec-3600*jam)
    menit = int(detik/60)
    detik = int(detik-60*menit)
    print("%.2d:%.2d:%.2d"%(jam, menit, detik))
else:
    hari  = int(sec/86400)
    jam   = int((sec-hari*86400)/3600)
    menit = int(((sec-hari*86400)-jam*3600)/60)
    detik = int(((sec-hari*86400)-jam*3600)-(menit*60))
    print("%d hari %.2d:%.2d:%.2d"%(hari, jam, menit, detik))